builtin/mainmenu/tab_settings.lua

```diff
@@ -247,7 +218,6 @@ local function formspec(tabview, name, tabdata)
 		"dropdown[0.25,3.2;3.3;dd_leaves_style;" .. leaves_style[1][1] .. ";"
 				.. getLeavesStyleSettingIndex() .. "]" ..
 		"box[3.75,0;3.75,3.45;#999999]" ..
-		"box[3.75,0;3.75,4.7;#999999]" ..
```

Comment:
Wrong. The **placement** is the 1st and 2nd number (i.e. `3.75, 0`). The **size** of the box is the 3rd and 4th number (i.e. `3.75,3.45`).

-> Two boxes overlay at each other, but one is extending downward. That's a mistake.